content/en/docs/concepts/containers/container-runtime-interface.md

```diff
@@ -0,0 +1,25 @@
+
+---
+reviewers:
+- dchen1107
+- derekwaynecarr
+title: Container Runtime Interface
+content_type: concept
+weight: 20
+---
+
+<!-- overview -->
+
+{{< feature-state for_k8s_version="v1.20" state="beta" >}}
```

Comment:
What does this represent? Is this that Kubernetes supports the CRI, as a beta Kubernetes feature? Or that Kubernetes supports using [some set of versions of] the CRI API?

Maybe the CRI support in Kubernetes is beta _and_ CRI itself is beta?